worker/worker.go

```diff
@@ -1487,14 +1489,15 @@ func (w *worker) scanHost(ctx context.Context, timeout time.Duration, hostKey ty
 			HostKey:    hostKey,
 			PriceTable: pt,
 			Subnets:    subnets,
-			Success:    isSuccessfulInteraction(err),
+			Success:    err == nil,
```

Comment:
I think this is incorrect, if you look at the update:

```sql
UPDATE hosts SET
		...
		recent_downtime = CASE WHEN `success` ... END,
		recent_scan_failures = CASE WHEN `success` ... END,
		downtime = CASE WHEN `success` ... END,
		uptime = CASE WHEN `success` ... END,
		...
		successful_interactions = CASE WHEN `success` ... END,
		failed_interactions = CASE WHEN `success` ... END,
	`)
```

So for insufficient funds/balance errors we'd be considering it a scan failure, failed interaction, add to downtime etc.